package.json

```diff
@@ -36,6 +36,7 @@
   "devDependencies": {
     "@travi/eslint-config-travi": "1.6.18",
     "commitlint-config-dsmjs": "1.0.3",
+    "core-js": "2.5.7",
```

Comment:
Otherwise:

> Error: Cannot find module 'core-js/modules/es6.array.filter'

Not sure why yarn handles this more cleanly 🤷‍♂️

Reply:
See gatsbyjs/gatsby#7862 (although it appears to be some type of babel 7 issue?)

Reply:
@travi I wonder if it's because some things are using babel-core (babel 6) and others are on babel 7? My guess is npm is de-duping too aggressively, and perhaps it was using an (older) version of core-js that didn't contain that package? Not sure. This isn't a huge change either way.

Reply:
i havent converted any of our projects to babel 7 yet, but it is becoming more of a pain to avoid it, so we're overdue to get over that hurdle. im at least aware of [babel-bridge](https://github.com/babel/babel-bridge), which is supposed to help handle things that are still set to use `babel-core`, so maybe that could help here?